locals.tf

```
@@ -7,6 +7,9 @@ locals {
   install_confluence = contains([for o in var.products : lower(o)], "confluence")
   install_bamboo     = contains([for o in var.products : lower(o)], "bamboo")
 
+  # If Bitbucket is the only product to install then we don't need to create shared home
+  share_home_size = local.install_bitbucket && length(var.products) == 1 ? "0Gi" : "5Gi"
```

I think the `0Gi` might be misleading.
 ```suggestion
  share_home_size = local.install_bitbucket && length(var.products) == 1 ? null : "5Gi"
```

Maybe even better to turn this into boolean:
```
should_create_efs = local.install_bitbucket && length(var.products) == 1 ? false : true
```

---

I prefer to stick to `null` as we should fulfill the validation condition for `share_home_size` anyway.